javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -680,6 +681,40 @@ public static Tuple0 tuple() {
         return Option.nothing();
     }
 
+    /**
+     * Alias for {@link Try#of(CheckedSupplier)}
+     *
+     * @param <T>      Component type
+     * @param supplier A checked supplier
+     * @return {@code Success(supplier.get())} if no exception occurs, otherwise {@code Failure(throwable)} if an
+     * exception occurs calling {@code supplier.get()}.
+     */
+    public static <T> Try<T> Try(CheckedSupplier<? extends T> supplier) {
```

Comment:
hmm, i don't find it intuitive that we've uppercased only this one (for obvious reasons, thouh).
@danieldietrich, how should we proceed in this case?